docs/content/misc/math-expr.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: doc_page
+---
+
+This expression language supports following operators (listed in increasing order of precedence).
```

Comment:
Precedence is generally shown in decreasing order.
There's a missing "the" in front of "following operators".